day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,23 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(toppings)
+    puts "Add #{toppings}"
```

Comment:
For this and the two below methods, you're printing your arguments, but how would you leverage those arguments to update your instance variables?